### PR TITLE
Update bookings-in-outlook.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a new section that talks about how to create a bookings with me page as it was not mentioned in the document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9650

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -202,6 +202,13 @@ Use the **Get-CASMailbox** and **Set-CASMailbox** commands to check user status
    Set-CASMailbox -Identity adam@contoso.com -EwsApplicationAccessPolicy EnforceBlockList -EWSBlockList @{Add="MicrosoftOWSPersonalBookings"}
    ```
 
+### Create Bookings with me
+
+1. Open [outlook on the web](https://go.microsoft.com/fwlink/p/?LinkID=402333)
+2. Click on **Calendar**
+3. Click on **Create bookings page** link  that appears in your calendar to create Bookings with me.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
1. Open [Outlook on the web](https://go.microsoft.com/fwlink/p/?LinkID=402333).
2. Select **Calendar**
3. Select the **Create bookings page** link that appears in your calendar to create Bookings with me.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Hi @msbemba, I am not sure but shouldn't this section rather be inserted right after the "Prerequisites for using Bookings with me" section?

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 2]
```diff
@@ -202,6 +202,12 @@ Use the **Get-CASMailbox** and **Set-CASMailbox** commands to check user status
    Set-CASMailbox -Identity adam@contoso.com -EwsApplicationAccessPolicy EnforceBlockList -EWSBlockList @{Add="MicrosoftOWSPersonalBookings"}
    ```
 
+### Create Bookings with me
+
+1. Open [Outlook on the web](https://go.microsoft.com/fwlink/p/?LinkID=402333).
+2. Select **Calendar**
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
2. Select **Calendar**.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
One more, please.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako it is not merged yet. Please review.

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder.

**Pull request merged**